### PR TITLE
ClayListGroup

### PR DESCRIPTION
**Pull request by carloslancha**
https://issues.liferay.com/browse/LEXI-50

Missing features:

- List element as a notification. Markup to be implemented. https://github.com/liferay/clay/issues/386

### CODE REVIEW

**Comment by carloslancha** [comment]
@jbalsas if this is merged remember that we need to update the portal ClaySticker taglib to fit with new flat API.

**Comment by matuzalemsteles** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: packages/clay-list-group/src/ClayListGroup.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+				'success',
+				'warning',
+			]).value('secondary'),
+			selected: Config.bool().value(false),
+			title: Config.string().required(),
+			url: Config.string(),
+		})
+	).required(),
+
+	label: Config.string(),
```
Comment (matuzalemsteles) [inline packages/clay-list-group/src/ClayListGroup.js]: Missing JSDOC.

File: packages/clay-list-group/src/ClayListGroup.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+	labelStyle: Config.oneOf([
```
Comment (matuzalemsteles) [inline packages/clay-list-group/src/ClayListGroup.js]: Missing JSDOC.

File: packages/clay-sticker/src/ClaySticker.js [hunk 3]
```diff
@@ -30,13 +30,10 @@ ClaySticker.STATE = {
 	 * Render ClayIcon in the ClaySticker element.
 	 * @instance
 	 * @memberof ClaySticker
-	 * @type {?Object|undefined}
+	 * @type {?string|undefined}
 	 * @default undefined
 	 */
-	icon: Config.shapeOf({
```
Comment (matuzalemsteles) [inline packages/clay-sticker/src/ClaySticker.js:36]: Hey @carloslancha, are we separating the `icon` to make it easier to use in JSP or to make it clearer?

**Review by matuzalemsteles — CHANGES_REQUESTED** [review]

File: packages/clay-sticker/src/ClaySticker.js (hunk 3 above)
Comment (carloslancha) [reply packages/clay-sticker/src/ClaySticker.js:36]: Both, we'll probably need to change some more API with that purpose in the next episode

File: packages/clay-list-group/src/ClayListGroup.js (hunk 1 above)
Comment (carloslancha) [reply packages/clay-list-group/src/ClayListGroup.js]: In fact I forgot to remove this property, it's inside the items shape, not here.

File: packages/clay-list-group/src/ClayListGroup.js (hunk 2 above)
Comment (carloslancha) [reply packages/clay-list-group/src/ClayListGroup.js]: In fact I forgot to remove this property, it's inside the items shape, not here.

**Review by matuzalemsteles — APPROVED** [review]

**Comment by matuzalemsteles** [comment]
Merged, thanks!

**Pull request merged**